rolling/distribution.yaml

```diff
@@ -3911,6 +3911,17 @@ repositories:
       url: https://github.com/ros2/rosidl_typesupport_gurumdds.git
       version: master
     status: developed
+  rot_conv_lib:
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/ros2-gbp/rot_conv_lib-release.git
+      version: 1.0.2-1
+    source:
+      type: git
+      url: https://github.com/ros2-gbp/rot_conv_lib-release.git
```

Comment:
@ijnek hmm, `source` should point to https://github.com/AIS-Bonn/rot_conv_lib, the actual source repository.

Reply:
Also, I've noticed that this is a pure CMake package, yet https://github.com/ros2-gbp/rot_conv_lib-release/blob/master/rolling/package.xml (incorrectly) indicates it's an `ament_cmake` package. 

Your `package.xml` needs an update.

Reply:
> @ijnek hmm, `source` should point to https://github.com/AIS-Bonn/rot_conv_lib, the actual source repository.

I've been using ``mavlink`` as an example and they have the release repository's patched branch as the source directory. I don't know if having the source entry point to the unpatched original repository is a good thing.

Reply:
> Also, I've noticed that this is a pure CMake package, yet https://github.com/ros2-gbp/rot_conv_lib-release/blob/master/rolling/package.xml (incorrectly) indicates it's an `ament_cmake` package.
> 
> Your `package.xml` needs an update.

Didn't know about this. I'll fix this up.

Reply:
> Also, it doesn't look like your package installs anything. The resulting binary package will be empty.

Today, I tried building the patch branch from source and using it with no success. Perhaps I have to add more patches to the CMakeLists.txt for this package to work at all.

Again, this is missing from the bloom documentation so I'll add it to https://github.com/ros2/ros2_documentation/pull/2375

Reply:
@hidmic I've added patches to install the package correctly, and changed the ``source`` to point to the code repo in #32670